graylog2-web-interface/src/components/graylog/styles/buttonStyles.js

```diff
@@ -10,15 +10,14 @@ const buttonStyles = ({ colors, active, utility }) => {
     const darken125 = utility.darken(color, 1.25);
 
     return css`
-    && {
```

Comment:
This whole file should be gone. Must've gotten mixed in with a merge somewhere. We can rely on `components/graylog/styles/button.jsx`